packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -4737,37 +4737,79 @@ QUnit.module('reset', () => {
     });
 
+    let screen;
+    const value = 'VINET';
+    const text = 'Vins et alcools Chevalier (France)';
+    const formOptions = {
+        formData: { CustomerID: value },
+        screenByWidth: function() { return screen; },
+        colCountByScreen: {
+            sm: 1,
+            lg: 2
+        },
+        items: [
+            {
+                itemType: 'tabbed',
+                tabPanelOptions: {
+                    deferRendering: false
+                },
+                tabs: [
+                    {
+                        title: 'Tab 1',
+                        colCount: 1,
+                        items: [
+                            {
+                                itemType: 'simple',
+                                cssClass: 'test-ddbox',
+                                dataField: 'CustomerID',
+                                editorOptions: {
+                                    displayExpr: 'Text',
+                                    valueExpr: 'Value',
+                                    showClearButton: true,
+                                    dataSource: [{ Value: value, Text: text }],
+                                },
+                                editorType: 'dxDropDownBox',
+                            },
+                        ]
+                    },
+                    {
+                        title: 'Tab 2',
+                        colCount: 1,
+                        items: [
+                            {
+                                itemType: 'simple',
+                                dataField: 'ShipCountry',
+                                colSpan: 1
+                            },
+                        ]
+                    }
+                ]
+            }
+        ]
+    };
+
     QUnit.test('DropDownBox should not lose its value if form resized (T1196835)', function(assert) {
-        let screen = 'lg';
+        screen = 'lg';
 
-        const value = 'VINET';
-        const text = 'Vins et alcools Chevalier (France)';
-        const $form = $('#form').dxForm({
-            formData: { CustomerID: value },
-            screenByWidth: function() { return screen; },
-            colCountByScreen: {
-                sm: 1,
-                lg: 2
-            },
-            items: [
-                {
-                    itemType: 'simple',
-                    cssClass: 'test-ddbox',
-                    dataField: 'CustomerID',
-                    editorOptions: {
-                        displayExpr: 'Text',
-                        valueExpr: 'Value',
-                        showClearButton: true,
-                        dataSource: [{ Value: value, Text: text }],
-                    },
-                    editorType: 'dxDropDownBox',
-                },
-            ]
-        });
+        const $form = $('#form').dxForm(formOptions);
+        const $input = $form.find(`.test-ddbox .${EDITOR_INPUT_CLASS}`);
+
```

Comment:
nothing is focused => I don't see a reason how value can be lost.
Is input focusing missing here?